README.md

```diff
@@ -30,10 +30,14 @@ For more information see the [product backlog](https://github.com/wmde/phragile/
 
 * PHP 5.5 or later
 * MySQL, SQLite or PostgreSQL
+* Phabricator ( see below )
 
 ### Preparation
 
 * [Install Phabricator](https://secure.phabricator.com/book/phabricator/article/installation_guide/)
+    
+    Phragile >=2.0.0 needs Phabricator >=2016 Week 8
```

Comment:
I would see the readme should always say something about the latest version requirements and the rest will go and remain in the release notes